tests/test_canonicalise.py

```diff
@@ -192,6 +192,13 @@ def test_canonicalises_to_expected(schema, expected):
         ([{"type": "null"}, {"enum": [None, True]}], {"const": None}),
         ([{"type": "null"}, {"type": ["null", "boolean"]}], {"const": None}),
         ([{"type": "integer"}, {"maximum": 20}], {"type": "integer", "maximum": 20}),
+        (
+            [
+                {"type": "integer", "mutlipleOf": 3},
+                {"type": "integer", "multipleOf": 6},
+            ],
+            {"type": "integer", "multipleOf": 6},
+        ),
```

Comment:
```suggestion
        ([{"mutlipleOf": 4}, {"multipleOf": 6}], {"multipleOf": 12}),
```

would check that we're not only handling `multipleOf` values which are multiples of each other, nor simply taking the product of all the options.